frontend/src/libs/components/sidebar/styles.module.scss

```diff
@@ -1,6 +1,10 @@
 .image {
-  width: 56px;
-  height: 56px;
+  width: 59px;
+  height: 59px;
+}
+
+.image img {
```

Comment:
Do not use global tags in styles. Rule B10 in quality criteria

Reply:
@fshabanov I updated the code, thanks for your review.